src/instruments/src/EFB/Dashboard/Widgets/WeatherWidget.tsx

```diff
@@ -104,6 +109,10 @@ const WeatherWidget = (props: WeatherWidgetProps) => {
             });
     }
 
+    if(baroType === "AUTO") {
+        baroType = getBaroTypeForAirport(props.icao);
+    }
+
```

Comment:
Suggestion: Consider adding this code and using it below to get around the nested ternary warning.

```
const BaroValue = () => {
        if (baroType === 'IN HG') {
            return (
                <>
                    {metar.barometer.hg.toFixed(2)}
                    {' '}
                    inHg
                </>
            );
        }
        return (
            <>
                {metar.barometer.mb.toFixed(0)}
                {' '}
                mb
            </>
        );
    };
```